app/src/main/java/space/taran/arknavigator/mvp/presenter/adapter/FoldersGridPresenter.kt

```diff
@@ -34,7 +34,13 @@ class FoldersGridPresenter(
 
     fun init(paths: List<Path>) {
         val folder = findLongestCommonPrefix(paths)
-        frames.addLast(Frame(folder, paths))
+
+        val (directories, files) = listChildren(folder)
```

Comment:
Variable `paths` must be used or refactored-out.

Reply:
The paths parameter is already used to create folder variable and then get contain of that folder.

Reply:
It isn't used in new version of the code. It seems to be computed again using `listChildren`.

Reply:
When user click in add(+) icon in folders list fragment, this `fun init(paths: List<Path>) ` method is called and `paths` has only one item and it is parent directory then `listChildren` get contain of `paths` item.